packages/shell/src/data-access/shell-providers.tsx

```diff
@@ -41,14 +41,3 @@ export function ShellProviders({ children }: ShellProviderProps) {
     </PersistQueryClientProvider>
   )
 }
-
```

Comment:
I added superjson and we should use this when we want to JSON.stringify BigInt or other things like Date